quetz/basic_frontend/index.html

```diff
@@ -51,6 +51,7 @@ <h1>Quetz</h1>
 <a href="/auth/github/login" style="margin-right: 100px" id="github_login">login with github</a>
 <a href="/auth/gitlab/login" style="margin-right: 100px" id="gitlab_login">login with gitlab</a>
 <a href="/auth/google/login" style="margin-right: 100px" id="google_login">login with google</a>
+<a href="/auth/azuread/login" style="margin-right: 100px" id="azuread_login">login with AzureAD</a>
 <a href="/auth/github/revoke" style="margin-right: 100px" id="revoke">revoke github</a>
 <a href="/auth/gitlab/revoke" style="margin-right: 100px" id="revoke">revoke gitlab</a>
 <a href="/auth/google/revoke" style="margin-right: 100px" id="revoke">revoke google</a>
```

Comment:
woudl be good to add the corresponding `revoke AzureAd` button here

Reply:
I added a revoke button, but its not working. Im getting the same error when clicking the github revoke button:

Server side:
```
"GET /auth/azuread/revoke HTTP/1.1" 422 Unprocessable Entity
```

Client side:
```
{"detail":[{"loc":["query","request"],"msg":"field required","type":"value_error.missing"}]}
````

Any idea?

Reply:
weird. looks like the endpoint expects a "request" field. 

<img width="680" alt="Screenshot 2021-09-08 at 10 38 16" src="https://user-images.githubusercontent.com/885054/132476053-f7044f4d-bab9-4ef9-b025-a7482e10f660.png">

Reply:
PS you can find this under localhost:8000/docs.

Will need to dig into what the contents for that request should be.